org.adempiere.ui.zk/WEB-INF/src/web/theme/default/css/fragment/drag-drop-attachment.css.dsp

```diff
@@ -0,0 +1,14 @@
+<%-- CSS for Drag & Drop on Attachment --%>
+
+.attachment-drag-entered {
+    border: 5px dashed #3fb900;
+}
+
+.drop-progress-meter
+{
+	width:100% !important;
+}
+
+.z-progressmeter-image {
+	background: linear-gradient(to bottom, #1eff00 0%, #1a6b18 100%);
+}
```

Comment:
for css changes, please also update the "idempiere.theme.default" timestamp in lang-addon.xml